src/components/incidenttable/FilteredIncidentTable.tsx

```diff
@@ -235,7 +237,7 @@ const FilteredIncidentTable: React.FC<FilteredIncidentsTablePropsType> = ({
         paginationComponent={paginationComponent}
       />
       <p>
-        Last refreshed {lastRefresh === undefined ? "never" : lastRefresh.toLocaleTimeString()}, {autoUpdateText}
+        Last refreshed {lastRefresh === undefined ? "never" : formatTimestamp(lastRefresh).slice(11)}, {autoUpdateText}
```

Comment:
Why do you use slice(11) here?

Reply:
I'm not using the entire format. Microseconds are unwanted,as is timezone.